bin/run_all.sh

```diff
@@ -35,7 +35,7 @@ for benchmark in `cat $root_dir/conf/benchmarks.lst`; do
     if [ $? -ne 0 ]
     then
 	echo "ERROR: ${benchmark} prepare failed!"
-        continue
+        exit $?
```

Comment:
`$?` here is not the result of prepare.sh, right? We may need to save the result first like `result=$`?
Can you please test it by failing one prepare.sh and see Travis's result?